src/scheduler/TaskScheduler.php

```
@@ -1,189 +1,194 @@
+
+    /**
+     * Check if a task is scheduled for a specific player.
+     */
+    public function isScheduledFor(Player $player): bool {
+        return isset($this->playerTasks[$player->getUniqueId()->toString()]);
+    }
+
+    /**
+     * Schedule a task for a player and store the reference.
+     */
+    public function scheduleForPlayer(Player $player, Task $task, int $ticks = 20): TaskHandler {
```

PHPStan (2.1.15)

Method pocketmine\scheduler\TaskScheduler::scheduleForPlayer() return type with generic class pocketmine\scheduler\TaskHandler does not specify its types: TTask


(missingType.generics)